### DISCUSSION TITLE
Redirect URI error

### DISCUSSION THREAD

**Discussion by TheRealStriff** [Barbarur/NovaPoint#11] [Q&A] [answered]
Tested this, and getting a URL redirect error.
(Similar to what we were getting with Nintex)

Sorry, but we’re having trouble with signing you in.

AADSTS50011: The redirect URI 'http://localhost:63958' specified in the request does not match the redirect URIs configured for the application 'XXXXXX'. Make sure the redirect URI sent in the request matches one added to your application in the Azure portal

**Comment by Barbarur** [comment]
Hello @TheRealStriff ,

Did you add the redirect URI *http://localhost/* for Mobile and Desktop applications on your Azure app registration? It seems that's missing.

**Reply by TheRealStriff** [reply, marked as answer]
Thanks.
I removed the App registration and did it again.
Working now, I thought I put the URI redirect in, but must have missed this the first time.
Thanks again, testing in Dev environment and working well.
Very much appreciate it (using for Version deletions)
